src/phases.hpp

```diff
@@ -0,0 +1,24 @@
+// Copyright 2018 Chia Network Inc
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//    http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef SRC_CPP_PHASES_HPP_
+#define SRC_CPP_PHASES_HPP_
+
+#pragma once
+enum phase_flags {
```

Comment:
you could specify the underlying type to match here, `enum phase_flags : uint8_t {`

Reply:
Done :)